sdk/typescript/entrypoint/load.ts

```diff
@@ -79,36 +119,63 @@ export function loadPropertyType(
  * Note: The JSON.parse() is required to remove extra quotes
  */
 export async function loadArg(
-  value: string,
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
These line comments are littered everywhere and make the code harder to read. Better to add them to eslint config and only use the inline comments sparsely on an exception to a rule.

Reply:
I could disable it for the file! Will think about it in the next PR